snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/Emitter.java

```diff
@@ -224,6 +226,17 @@ public EmitterBuilder timeUnit(TimeUnit timeUnit) {
             return this;
         }
 
+        /**
+         * @param client An OkHttp client that will be used in the emitter, you can provide your
+         *               own if you want to share your Singleton client's interceptors, connection pool etc..
+         *               ,otherwise a new one is created.
```

Comment:
should we mention the configuration we are overwriting below like connect and read timeout?